src/AdminPage.js

```diff
@@ -68,11 +68,12 @@ const AdminChild = () => {
   return (
     <>
       <Row>
-        <h1>{finalAdminEvent.eventname}</h1>
-      </Row>
-      <Row>
-        {successMessage && <Alert variant="success">{successMessage}</Alert>}
+        <h1>{finalAdminEvent.eventname ?? "Untitled event"}</h1>
+        <h2>{finalAdminEvent.eventDesc ?? ""}</h2>
```

Comment:
When I pulled this branch in to test, it isn't returning the event description. We also should avoid having an empty tag so something like this might be a better option:
`{finalAdminEvent.eventDesc ? <h2>{finalAdminEvent.eventDesc}</h2> : ''}`

Reply:
Many of the events have blank descriptions, so yeah, that makes sense